src/App/Fossa/VSI/Types.hs

```diff
@@ -77,9 +77,12 @@ instance ToJSON Locator where
 instance FromJSON Locator where
   parseJSON = withObject "Locator" $ \obj -> do
     Locator
-      <$> obj .: "fetcher"
-      <*> obj .: "package"
-      <*> obj .: "revision"
+      <$> obj
+        .: "fetcher"
+      <*> obj
+        .: "package"
+      <*> obj
+        .: "revision"
```

Comment:
Auto formatted

Reply:
If you use this fixities directive in `fourmolu.yaml` a better formatting should be allowed by the HLS plugin:
```
fixities:
    - infixr 0 $
    - infixl 3 <|>
    - infixr 3 &&
    - infix 4 ==
    - infixl 4 <$>, <*>
```
The relevant parts are the ones for `$`, `<$>`, `<*>` but I've added the entire directive. For this `FromJSON` instance in particular you'll have to put the formatting back manually but after that `fourmolu` shouldn't touch it. 

Sorry that this happened!